apiv2/db_import/tests/populate_db.py

```diff
@@ -396,6 +399,29 @@ def populate_tiltseries(session: sa.orm.Session) -> Tiltseries:
         tilting_scheme="unknown",
         data_acquisition_software="unknown",
     )
+    frame = Frame(
+        run_id=RUN1_ID,
+        id=FRAME_ID,
+        acquisition_order=0,
+        deposition_id=DEPOSITION_ID1,
+        s3_frame_path="s3://test-public-bucket/30001/RUN1/Frames/frame1",
+        https_frame_path="https://foo.com/30001/RUN1/Frames/frame1",
+    )
+    session.add(frame)
+    per_section_parameters = PerSectionParameters(
```

Comment:
Is there additional changes to the tiltseries test, to validate the updates to this record? 

As the PerSectionParameter has been configured for cleaning up stale records, should we add a stale record to test that as well?